pkg/api/helper/helpers.go

```diff
@@ -594,3 +594,27 @@ func PersistentVolumeClaimHasClass(claim *api.PersistentVolumeClaim) bool {
 
 	return false
 }
+
+// GetTopologyConstraintsFromAnnotations gets the json serialized topology constraints data from PersistentVolume.Annotations
+// and converts it to the []TopologyConstraint type in api.
+// TODO: remove when alpha support for topology constraints is removed
```

Comment:
For beta, we will still have annotations, so I updated the comment to say to update the function when we move to beta.